pom.xml

```diff
@@ -312,12 +312,13 @@
         <version.io.rest-assured>3.0.6</version.io.rest-assured>
         <version.io.smallrye.smallrye-common>1.5.0</version.io.smallrye.smallrye-common>
         <version.io.smallrye.smallrye-config>2.0.2</version.io.smallrye.smallrye-config>
-        <version.io.smallrye.smallrye-fault-tolerance>4.3.2</version.io.smallrye.smallrye-fault-tolerance>
         <version.io.smallrye.smallrye-health>3.0.0-RC2</version.io.smallrye.smallrye-health>
+        <version.io.smallrye.smallrye-fault-tolerance>5.0.0-RC3</version.io.smallrye.smallrye-fault-tolerance>
```

Comment:
The 5.0 final is out now that we have been waiting for.

Reply:
Updated.